package.json

```diff
@@ -55,7 +55,7 @@
     "@opentelemetry/resources": "1.3.1",
     "@opentelemetry/sdk-trace-node": "1.3.1",
     "@opentelemetry/tracing": "0.24.0",
-    "@trussworks/react-uswds": "3.0.2",
+    "@trussworks/react-uswds": "https://github.com/trussworks/react-uswds#main",
```

Comment:
this requires USWDS 2.13.3 which has been merged to ReactUSWDS main but not released yet